packages/create-rainbowkit/generated-test-app/pages/_app.tsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
Was there a reason we needed this change?

Reply:
Yeah unless i change something in tsconfig.json:

![image](https://github.com/rainbow-me/rainbowkit/assets/53529533/f2cdab51-e3bb-4035-a95d-169207a50bb2)